app/src/main/java/com/danlls/daniel/todule_android/activities/TimeReminderFragment.java

```diff
@@ -0,0 +1,170 @@
+                        @Override
+                        public void onDismiss(DialogInterface dialogInterface) {
+                            timeViewHours.clearFocus();
+                        }
+                    });
+                    timePicker.show();
+                }
+            }
+        });
+
+        timeViewHours.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+            }
+
+            @Override
+            public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+            }
+
+            @Override
+            public void afterTextChanged(Editable editable) {
+                long date_time = myCalendar.getTimeInMillis();//utilise ca
+            }
+        });
+
+        timeViewHours.setText(myCalendar.get(Calendar.HOUR_OF_DAY) +":"+myCalendar.get(Calendar.MINUTE));
+
+
+
+        return view;
+    }
+
+    private void ShowTos(String msg) {
+        Toast.makeText(getActivity(), "Reminder updated "+msg,
+                Toast.LENGTH_LONG).show();
+    }
+
+    public long getOnlyHoursAndMins(int dayHourMin,int t){
+        if (t == 1) return dayHourMin * 60 * 60 * 1000; // convert hours to millis
+        if (t == 2) return dayHourMin * 24 * 60 * 60 *1000; // convert days to millis
+        else return dayHourMin * 60 * 1000; // convert minutes to millis
+    }
+
+
+
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        menu.clear();
+        inflater.inflate(R.menu.menu_fragment_time_save, menu);
+        super.onCreateOptionsMenu(menu, inflater);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch(item.getItemId()) {
+            case R.id.action_save:
+                long timingInMillis = getOnlyHoursAndMins(Integer.parseInt(timeViewDays.getText().toString()),2)
+                        + getOnlyHoursAndMins(myCalendar.get(Calendar.HOUR_OF_DAY),1)
+                        + getOnlyHoursAndMins(myCalendar.get(Calendar.MINUTE),3);
+                MainActivity.timing = timingInMillis;
+                MainActivity.Tdays = Integer.parseInt(timeViewDays.getText().toString());
+                MainActivity.Thours = myCalendar.get(Calendar.HOUR_OF_DAY);
+                MainActivity.Tmins = myCalendar.get(Calendar.MINUTE);
+                SharedPreferences.Editor editor = sf.edit();
+                editor.putLong("timing", timingInMillis);
+                editor.putInt("hours", myCalendar.get(Calendar.HOUR_OF_DAY));
+                editor.putInt("minutes", myCalendar.get(Calendar.MINUTE));
+                editor.putInt("days", Integer.parseInt(timeViewDays.getText().toString()));
+                editor.commit();
+                ShowTos(timeViewDays.getText().toString()+" Days and "+timeViewHours.getText().toString());
```

Comment:
I think you missed the formatting for hour and minutes, toast is showing "Reminder updated 1 Days and 1:0"